web/.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
   "jsxSingleQuote": true,
-  "plugins": ["prettier-plugin-sort-json"],
+  "plugins": ["prettier-plugin-sort-json", "prettier-plugin-tailwindcss"],
```

Comment:
you don't _need_ this but it makes life a lot better when dealing with tailwind classes